src/app/home/navigation.component.ts

```diff
@@ -11,7 +11,17 @@ import { Router } from '@angular/router';
       <li><a href="#" class="km-logout" (click)="logoutClick()">LOGOUT</a></li>
       <li><a routerLink="/manager"><i class="material-icons">settings</i></a></li>
       <li *ngIf="roles.indexOf('_admin') === -1"><a routerLink="/users/profile/{{name}}"><mat-icon>person</mat-icon></a></li>
+      <li><a href="#"><mat-icon [matMenuTriggerFor]="notification">notifications</mat-icon></a></li>
     </ul>
+    <mat-menu #notification="matMenu">
+      <mat-list>
+        <mat-list-item> Notification 1 </mat-list-item>

```

Comment:
Each element should simply be a `mat-menu-item` (see https://material.angular.io/components/menu).  I think you can use a `div` rather than `button` to get the same effect without the odd spacing that the `mat-list` is creating.